packages/governance/docs/contractGovernance.puml

```diff
@@ -10,7 +10,7 @@ package "GovernedContract Vat" <<Rectangle>>  {
     }
 
     class GovernedContract {
-        <i>verifiable</i>: Governor, params
+        <i>verifiable</i>: Governor, params, governedApis
```

Comment:
I looked into replacing this with Mermaid.js so it wouldn't need a build step. Unfortunately it [doesn't support package in class digrams](https://github.com/mermaid-js/mermaid/issues/1052).